nui/src/main/java/org/terasology/nui/layouts/FlowLayout.java

```diff
@@ -98,4 +161,92 @@ public Vector2i getMaxContentSize(Canvas canvas) {
     public Iterator<UIWidget> iterator() {
         return contents.iterator();
     }
+
+    /**
+     * Retrieves the horizontal spacing between adjacent widgets in this {@code FlowLayout}.
+     *
+     * @return The spacing, in pixels
+     */
+    public int getHorizontalSpacing() {
+        return horizontalSpacing;
+    }
+
+    /**
+     * Retrieves the vertical spacing between adjacent widgets in this {@code FlowLayout}.
+     *
+     * @return The spacing, in pixels
+     */
+    public int getVerticalSpacing() {
+        return verticalSpacing;
+    }
+
+    /**
+     * Sets the horizontal spacing between adjacent widgets in this {@code FlowLayout}.
+     *
+     * @param spacing The spacing, in pixels
+     * @return This {@code FlowLayout}
+     */
+    public FlowLayout setHorizontalSpacing(int spacing) {
+        this.horizontalSpacing = spacing;
+        return this;
+    }
+
+    /**
+     * Sets the vertical spacing between adjacent widgets in this {@code FlowLayout}.
+     *
+     * @param spacing The spacing, in pixels
+     * @return This {@code FlowLayout}
+     */
+    public FlowLayout setVerticalSpacing(int spacing) {
+        this.verticalSpacing = spacing;
+        return this;
+    }
+
+    /**
+     * Whether the directional flow of this layout goes from left-to-right or right-to-left.
+     * <p>
+     * If false, the children are laid out from left-to-right and aligned at the left border of the canvas. If true, the
+     * children are laid out right-to-left and aligned at the right border of the canvas.
+     * <p>
+     * This toggle can be set programmatically or in {@code .ui} files that use the Flow layout.
+     *
+     * @return whether the children are laid out right-to-left or aligned to the right
+     */
+    public boolean isRightToLeftAlign() {
+        return rightToLeftAlign.get();
+    }
+
+    /**
+     * Set whether the directional flow of this layout goes from left-to-right or right-to-left.
+     * <p>
+     * The children are laid out from left-to-right by default (false), aligned at the left border of the canvas. If
+     * this toggle is explicitly enabled (true) the children are laid out right-to-left, aligned at the right border of
+     * the canvas.
+     * <p>
+     * This toggle can be set programmatically or in {@code .ui} files that use the Flow layout.
```

Comment:
> This toggle can be set in {@code .ui} files that use the Flow layout.

How?

Reply:
for instance, via this method...I feel this is too much duplicated docs.... 🤔